src/features/embalm/stepContent/components/BundlrAlertMessage.tsx

```diff
@@ -2,18 +2,29 @@ import { SarcoAlert } from 'components/SarcoAlert';
 import { useSelector } from 'store/index';
 import { useGetBalance } from '../hooks/useGetBalance';
 import { useUploadPrice } from 'features/embalm/stepNavigator/hooks/useUploadPrice';
+import { useBundlr } from '../hooks/useBundlr';
+import { ethers } from 'ethers';
 
 export function BundlrAlertMessage() {
   const file = useSelector(x => x.embalmState.file);
   const { balance } = useGetBalance();
   const { uploadPrice } = useUploadPrice();
+  const { isFunding } = useBundlr();
+  const { balanceOffset } = useSelector(x => x.bundlrState);
 
   if (!file) {
     return <SarcoAlert status="warning">Upload a payload to get the upload price.</SarcoAlert>;
   }
 
   if (balance.lt(uploadPrice)) {
-    return <SarcoAlert status="warning">You need to add funds to Bundlr.</SarcoAlert>;
+    return isFunding || !balanceOffset.eq(ethers.constants.Zero) ? (
+      <SarcoAlert status="info">
+        Bundlr is currently being funded. You can continue to modify your sarcophagus during this
+        process as well as add more funds if necessary. You will be notified when funding completes.
```

Comment:
Is this true? Not sure that any notification occurs when the balance fully updates --- but if so, ignore this comment, as that is sweet if we have that.

Reply:
That isn't true lmao. I just copied the text from the designs without reading it entirely. It might be worth looking into if we can make this true though.